core/profiles/standard/standard.install

```diff
@@ -199,20 +202,22 @@ function standard_install() {
       'body' => array(
         LANGUAGE_NONE => array(
           array(
-            'value' => st('<h3>Welcome to your new site.</h3>
+            'value' => st('
             <p>A great place to start configuring your new site is with your home page layout.</p>
             <ul>
-              <li>You can <a href="node/1/edit">edit this message</a> by looking for the "Home page" listed under the Content menu.</li>
-              <li>You can add or remove blocks on this page layout at <a href="admin/structure/layouts">admin/structure/layouts.</a></li>
+              <li>You can <a href="node/1/edit">edit this message</a> by looking for the "Home page content" listed under the Content menu.
```

Comment:
Are we 100% sure that this node will always have ID `1`?

Reply:
We control the order in which the nodes are created during installation, so I can't see any way that it would be different. But, I'm not 100% sure of anything.

Reply:
More nodes may be added to the installation profile in the future, or their order may change. Also, if people copy/paste code to create their own profiles, and the order in their profiles is different, then the IDs won't be matching the actual nodes.

I think it'd be safer to use something like `backdrop_lookup_path()` to get the internal path of the node that was created (just a quick though - haven't fully fleshed that out).

Reply:
...I think that `$page->id()` might work actually, but since this is a new node, it's not saved yet, so there's no node ID available. So for such a solution to work, you'd need to save the node in 2 steps:
1. add nothing in the body field in the initial `$page->save();`
2. use `$page_id = $page->id();` after it is saved.
3. edit the page to add the content of the body field (now the actual ID is available to you)

Reply:
..something like this:

```php
  $page = new Node(
    array(
      'title' => st('Home page content'),
      'body' => array(),
      'uid' => 1,
      'status' => 1,
      'promote' => 0,
      'type' => 'promobox',
    )
  );
  $page->save();
  $page_id = $page->id();
  $page_edit_link = "node/$page_id/edit";
  $page->body = array(
    LANGUAGE_NONE => array(
      array(
        'value' => st('blah blah ... <a href="@link">edit this message</a> ...', array('@link' => $page_edit_link)),
        'format' => 'filtered_html',
      ),
    ),
  );
  $page->save();
```

Reply:
@stpaultim please check https://github.com/stpaultim/backdrop/pull/1 and consider it. I think it's more future-proof that way.